desmume/src/frontend/posix/gtk/graphics.ui

```diff
@@ -65,6 +65,7 @@
                   <item translatable="yes">Null</item>
                   <item translatable="yes">SoftRasterizer</item>
                   <item translatable="yes">OpenGL</item>
+				  <item translatable="yes">OpenGL ES</item>
```

Comment:
you seem to have mixed tabs and spaces here

Reply:
Fixed